alacritty/src/window.rs

```diff
@@ -307,7 +299,7 @@ impl Window {
     }
 
     #[cfg(windows)]
-    pub fn set_urgent(&self, _is_urgent: bool) {}
+    pub const fn set_urgent(&self, _is_urgent: bool) {}
```

Comment:
What's the purpose of this change?

Reply:
Any function that I can declare `const`, I do. But I just dumped the IR and can see that rustc isn't picking up the opportunity to inline the void return. I'll revert it.

Reply:
It should be optimized out entirely regardless.

Making this `const` is just confusing and unnecessary extra text.